ballerina/natives.bal

```diff
@@ -16,17 +16,17 @@
 
 import ballerina/jballerina.java;
 
-final StatisticConfig[] DEFAULT_GAUGE_STATS_CONFIG = [{ timeWindow: 600000, buckets: 5,
+isolated final StatisticConfig[] DEFAULT_GAUGE_STATS_CONFIG = [{ timeWindow: 600000, buckets: 5,
     percentiles: [0.33, 0.5, 0.66, 0.75, 0.95, 0.99, 0.999] }];
 
-final map<string> DEFAULT_TAGS = {};
+isolated final map<string> defaultTags = {};
```

Comment:
Where is `defaultTags` updated? If it is not updated at all, can be in-lined. If you really need a default (empty map), can have a constant instead - which is inherently final and immutable, therefore, can get rid of locks, isolated, clones, etc.

Reply:
defaultTags is not updated and it has been used as an empty map. I will use use a constant instead.